README.md

```diff
@@ -203,34 +203,119 @@ All official releases shall be in <code>master</code>. Any updates in between (i
 After a pull request has been open for over 30 days with no activity or response from the author, it'll be automatically marked as stale. We might fork your changes and merge the changes ourselves. Since GitHub tracks contributions by commits, you will be credited.
 </p>
 
-<h2 id="build-yourself">Go Build Yourself</h2>
+<h2 id="building-devicon">Building Devicon</h2>
 <p>
-    Feel free to follow these steps when you want to build the font
-    by yourself.
+Follow these steps to build the website and icons either locally or using <a href=https://www.gitpod.io>Gitpod.io</a>.
 </p>
-<h5>Prerequisites</h5>
-<p>Install <code>gulp</code> (and <code>gulp</code> plugins)</p>
+
+<h3>Table of contents</h3>
+<ol>
+    <li><a href="#using-gitpod">Using Gitpod to build</a></li>
+    <li><a href="#installation">Installation (Linux/Windows)</a></li>
```

Comment:
Looks like the Linux and Windows installation processes are quite different, perhaps we should either just point to the official installation websites, or split this into two sections.

I would prefer if we just pointed to the installation site, since that would require less maintenance work from us in case the installation method changes. It could for example be as simple as:

### Install Python 3.8
https://www.python.org/downloads/

### Install Firefox
https://www.mozilla.org/en-US/firefox/new/

### Install Selenium
```bash
python3 -m pip install --upgrade pip && pip install selenium==4.1.0 requests==2.25.1
```